README.md

```diff
@@ -0,0 +1,156 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 HakcSummit 2023 <a name="about-project"></a>
+
+**HackSummit 2023** is a capstone project for HTML CSS and JavaScript module at microverse
+
+## 🛠 Built With <a name="built-with"> HTML CSS & Java Script</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML CSS & Java Script
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Presents the Home page for HackSummit 2023**
+- **Presents the About page for HackSummit 2023**
+- **Displays Speakers List with more and less button functionality**
+- **Displays Mobile Menu**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> This projcet is not deployed currently
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+### Usage
+
+To run the project open the index.html file in your web browser
+
+### Run tests
+
+No test are available
+
+### Deployment
+
+This project is not currently deployed
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] In this project we were suggested to deploy our website by using GitHub Pages. Kindly deploy your repository and make sure that all pages, links, and images work as expected, and remember to provide the deployment link in the README 🙏 

- [ ] Kindly as suggested in the project requirements, we have also to record a presentation video presenting the features of the project 🙏 
    - Please, the maximum length of the video should be 5 minutes.
    - in the video remember to include a clear and objective description of the project in professional English and the topic and purpose of the project.
    - The video contains a demo of the project, showing the pages in different screen sizes. 🙏 
    - Remember also to highlight a piece of the code that finds interesting or proud of making 👍
    - It is also suggested to add the links in the PR description to make it more professional 👍 

**Check the project requirements 👇**
![image](https://user-images.githubusercontent.com/102757126/226088280-0a9ba6d3-c3e6-4232-8965-28c59837a831.png)